sdk/identity/azure-identity/azure/identity/_credentials/environment.py

```diff
@@ -73,6 +76,9 @@ def __init__(self, **kwargs):
                 **kwargs
             )
         elif all(os.environ.get(v) is not None for v in EnvironmentVariables.USERNAME_PASSWORD_VARS):
+            if EnvironmentVariables.AZURE_CLIENT_SEND_CERTIFICATE_CHAIN in os.environ and "send_certificate_chain" not in kwargs:
+                send_chain = os.environ[EnvironmentVariables.AZURE_CLIENT_SEND_CERTIFICATE_CHAIN].lower() == "true" or os.environ[EnvironmentVariables.AZURE_CLIENT_SEND_CERTIFICATE_CHAIN] == "1"
```

Comment:
```suggestion
                send_chain = os.environ[EnvironmentVariables.AZURE_CLIENT_SEND_CERTIFICATE_CHAIN].lower() in ["true", "1"]
```
is shorter and I expect not notably slower